extensions/amp-fit-text/0.1/test-e2e/test-amp-fit-text.js

```diff
@@ -43,10 +43,92 @@ describes.endtoend(
+      it('when box size increases font size should increase', async () => {
+        const contentButton = await controller.findElement('#test5_button');
+        const contentDiv = await selectContentDiv('test5');
+
+        // Button click increases amp-fit-text box size
+        const originalfontsize = parseInt(
+          await controller.getElementCssValue(contentDiv, 'font-size'),
+          10
+        );
+        await controller.click(contentButton);
+        // wait for the resizeobserver to recognize the changes
+        // 90ms chosen so that the wait is less than the throttle value for the resizeobserver.
+        await new Promise((resolve) => {
+          setTimeout(() => {
+            resolve();
+          }, 90);
+        });
+        const updatedfontsize = parseInt(
+          await controller.getElementCssValue(contentDiv, 'font-size'),
+          10
+        );
+
+        await expect(updatedfontsize).to.be.greaterThan(originalfontsize);
+      });
+
+      it('when box size decreases font size should decrease', async () => {
+        const contentButton = await controller.findElement('#test6_button');
+        const contentDiv = await selectContentDiv('test6');
+
+        // Button click decreases amp-fit-text box size
+        const originalFontSize = parseInt(
+          await controller.getElementCssValue(contentDiv, 'font-size'),
+          10
+        );
+        await controller.click(contentButton);
+        // wait for the resizeobserver to recognize the changes
+        // 90ms chosen so that the wait is less than the throttle value for the resizeobserver.
+        await new Promise((resolve) => {
+          setTimeout(() => {
+            resolve();
+          }, 90);
+        });
+        const updatedFontSize = parseInt(
+          await controller.getElementCssValue(contentDiv, 'font-size'),
+          10
+        );
+
+        await expect(updatedFontSize).to.be.lessThan(originalFontSize);
+      });
+    });
+
+    it.only('should recalculate font size after box size change', async () => {
+      debugger;
+      const contentButton = await controller.findElement('#test5_button');
+      const contentDiv = await selectContentDiv('test5');
+
+      const originalFontSize = parseInt(
+        await controller.getElementCssValue(contentDiv, 'font-size'),
+        10
+      );
+      const originalWidth = await controller.getElementCssValue(
+        contentDiv,
+        'width'
+      );
+
+      await expect(originalWidth).to.equal('100px');
+
+      await controller.click(contentButton);
+      const updatedFontSize = parseInt(
+        await controller.getElementCssValue(contentDiv, 'font-size'),
+        10
+      );
+      const updatedWidth = await controller.getElementCssValue(
+        contentDiv,
+        'width'
+      );
+      await expect(updatedWidth).to.equal('200px');
+
+      await expect(updatedFontSize).to.be.greaterThan(originalFontSize);
+    });
+
```

Comment:
I think this is the same as the first test above (there also shouldn't be an `only()`).

```suggestion
```